Janitor QoL, Vendor Updates, and some tweaks.

Owai-Seek opened the pull request:
## About The Pull Request

Updated Metal Shields Bounty (It was too hard for so little.)
Added Cable Coils and Screwdriver to Janibelt Whitelist
Added Cable Coils and Screwdriver to Janidrobe.
Added Leg Wraps to Clothesmate

## Why It's Good For The Game

Janitor Man Fix Lights. Also adding missing items to the clothing vendor.

While I was at it, updated some vendor contents because the stuff they unlocked was kinda niche or disappointing.

## Changelog
:cl:
add: leg wraps and sweaters to clothesmate
add: screwdriver and cable coil to janidrobe
add: screwdriver and cable coil to janibelt whitelist (for fixing/placing light fixtures)
add: monkey cube, syringe, enzyme, soy sauce, and cryoxadone to chef's vendor (contraband and premium)
add: add cracker, beans, honey bars, lollipops, chocolate coin, and spider lollipop to snack vendors (contraband and premium)
add: newspaper to loadout menu for bapping purposes
del: removed poppy pretzels from snack vendor premium
add: maid uniform (janimaid alt) to kinkmate.
tweak: moves gear harness from premium to normal stock in kinkmate
balance: re-balanced metal shield bounty
add: cryoxadone bottle (for use in chef vendor)
/:cl:

Owai-Seek committed:
Minor QoL and Wardrobe Updates

Trilbyspaceclone commented on code/modules/cargo/bounties/assistant.dm:
```diff
@@ -201,10 +201,10 @@
 	description = "NT is testing the effects of electricity on clowns wielding metal shields. We have clowns, and we have electricity. Send us the shields."
-	reward = 1400
-	required_count = 4
+	reward = 2300
```
raise this up to 5k for 2 imo

Owai-Seek replied:
I raised it for 2 for 3k. Cargo can print the tools, metal, and cables for this. You can easily get cable from bedsheets, but the leather is the thing that slows it down, which you can get from both ghetto botany and normal botany with enough effort.

Owai-Seek committed:
Shields Bounty is now 2 for 3000

Owai-Seek committed:
Added more items to vendors.

-Added stuff to the following:
clothing vendor
chef utensil vendor
snack vendors
kinkmate

Owai-Seek committed:
Added Newspaperr to Loadout

Putnam3145 committed:
Merge branch 'master' into tinytweaks
